spec/tools/fix_auth/auth_config_model_spec.rb

```diff
@@ -38,14 +40,10 @@
       expect(request).to be_changed
       new_options = YAML.load(request.options)
       expect(new_options[:dialog]['password::special'.to_sym]).to be_encrypted(pass)
```

Comment:
aren't we dropping to be encrypted?

I originally used be_encrypted because this was using an asymmetric cipher instead of a symmetric cipher. with that case, encrypted values were always different and a standard `eq` would not work.

Our current cipher produces the same output from the same input and eq works

Reply:
be_encrypted_version will be dropping, but be_encrypted still exists (even so, this whole PR works with the existing manageiq-password and doesn't require any changes on that side).